metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -142,23 +156,67 @@ public ObjectsImporter(
         this.taxonomyEventHandler = taxonomyEventHandler;
     }
 
-    public Study importStudy(String accession) throws Exception {
-        IMPORT_LOGGER.log(Level.INFO, "Importing study " + accession);
+    public Project importProject(String accession) throws Exception {
+        LOGGER.info("Importing project " + accession);
+        ProjectType projectType = retrieveType(accession, sraProjectXmlParser);
+        if (projectType == null) { return null; }
+        Project project = convertProject(projectType);
+
+        String studyAccession = projectType.getIDENTIFIERS().getSECONDARYIDArray(0).getStringValue();
```

Comment:
What worries me in that method is getSECONDARYID**Array.** Looks like it is possible for a project to have multiple studies (secondary IDs) linked to it after all. Sure, for the EVA use case it won't matter because it's always 1-to-1, but in general this doesn't appear to be the case

Reply:
This one was tough to figure out.

Assuming EGA will only import from DB, my understanding is that EGA will import analysis (and associated studies, but NO projects), and EVA will import projects (with studies and analysis), so I don't think getSECONDARYIDArray will make a problem here:

- **project**`.getSECONDARYIDArray` with several studies: can only have more than 1 elements for EGA, and EGA doesn't import projects.

- **project**`.getSECONDARYIDArray` with 0 studies: this too will only be called by EVA, so it will never have 0 studies.

- **study**`.getSECONDARYIDArray` with several projects: I think should never be more than 1 so I'll throw an exception. 

- **study**`.getSECONDARYIDArray` with 0 elements: In subsequent commits I allow importing a study without projects, as some tests import EGA studies that don't have projects. This code is also called from importAnalyses.